app/models/users/setting.rb

```diff
@@ -0,0 +1,56 @@
+module Users
+  class Setting < ApplicationRecord
+    self.table_name_prefix = "users_"
+
+    EDITORS = %w[v1 v2].freeze
+    FONTS = %w[serif sans_serif monospace comic_sans open_dyslexic].freeze
+    INBOXES = %w[open private].freeze
+    NAVBARS = %w[default static].freeze
+    THEMES = %w[default night_theme pink_theme minimal_light_theme ten_x_hacker_theme].freeze
+
+    MESSAGES = {
+      invalid_config_font: "%<value>s is not a valid font selection",
+      invalid_config_navbar: "%<value>s is not a valid navbar value",
+      invalid_config_theme: "%<value>s is not a valid theme",
+      invalid_editor_version: "%<value>s must be either v1 or v2"
+    }.freeze
+
+    before_validation :set_config_input
+
+    validates :user_id, presence: true
+    validates :config_font, inclusion: { in: FONTS + ["default".freeze], message: MESSAGES[:invalid_config_font] }
```

Comment:
Is there a reason "default" can't be in the `FONTS` constant (or an enum if you decide to make that change)?

Reply:
I added it 👍🏾

Reply:
@fdoxyz or @ludwiczakpawel any idea if there was a specific reason we did this? It seems related to [this PR](https://github.com/forem/forem/pull/9650).

Reply:
i don't wanna lie but to be honest, i can't quite remember the exact reason :( maybe @fdoxyz ?

Reply:
At the time (and still today) the database creates new users with `default` as the default value ([it's most of them](https://dev.to/admin/blazer/queries/348-users-with-default-font)). The logic can be summarized as:

> Replace _"default"_ (from `user.config_font` value)  with the actual font name set in `SiteConfig.default_font` in the Forem instance. Once the user updates their choice of font from their settings we just serve that choice (font name) unchanged.

#9650 allowed creators to customize the default font to be used (it used to be hardcoded), which was part of a design refactor so no major logic changes were introduced. The reason it was added to the validation in the User model must have been a validation error that bubbled up at some point. 

At the time I didn't want to add `default` to the `FONTS` constant because it was used to populate the User facing form [here](https://github.com/fdoxyz/forem/blob/master/app/views/users/_customization.html.erb#L23). This would mean users would see `default` as a valid option, and it would be the exact same font as the Forem's default font (i.e. `default` would be "sans serif font" and `sans serif` would also be "san serif font"). 

However, looking back to it maybe it's okay for users to be able to choose `default` font even if it results in this redundancy? I'm okay with either option, but it's important to keep in mind this side effect on the User settings form.

Reply:
Thanks so much for this clear explanation @fdoxyz. @msarit I vote for keeping it as is for now.